shared/general/persistent-storage.md

```diff
@@ -19,6 +18,23 @@ When using named volumes, note that:
 - If a device is moved to a new application, the old `/data` folder will be automatically purged.
 - During the build process, data added to a container directory that is configured to link to a named volume will be copied to the volume the first time it's created on the device.
 
+#### Using a Supervisor with a version >= v10.0.0
+
+Since v10.0.0 of balena-supervisor, volumes are no longer
```

Comment:
```suggestion
Since balena-supervisor v10.0.0, volumes are no longer automatically removed from disk when references to them are removed from an application's `docker-compose` file. This means that it's no longer possible for data to be lost due to the accidental rename of a volume. 

If you change volume names regularly, your device will now continue to retain all previous volumes from an application including their contents. To avoid this the supervisor API now provides an [endpoint to cleanup unreferenced volumes](https://www.balena.io/docs/reference/supervisor/supervisor-api/#cleanup-volumes-with-no-references). Additionally, it is possible to perform this action from the dashboard via the `Purge Data` action, found on the `Actions` tab for a device.

**Note:** volumes will continue to be removed automatically when moving a device between applications.
```